pipeline/relation_embedder/relation_embedder/src/test/scala/uk/ac/wellcome/relation_embedder/RelationEmbedderWorkerServiceTest.scala

```diff
@@ -51,17 +44,19 @@ class RelationEmbedderWorkerServiceTest
     * 1  2
     * |  |---
     * |  |  |
-    * b  c  d
+    * b  c  d*
     *    |
     *    |
     *    e
+    *
+    * *d is available online
```

Comment:
nit: maybe use a caret `^`? just because asterisk is already being used here.

Reply:
lol, intellij didn't like the caret and told me I was using incorrect wiki syntax. Used a dagger.